imports/lib/schemas/Announcement.ts

```diff
@@ -1,6 +1,6 @@
 import * as t from 'io-ts';
-import SimpleSchema from 'simpl-schema';
 import { BaseCodec, BaseOverrides } from './Base';
+import { Id } from './regexes';
```

Comment:
OOC, is there any built-in value `Id` that we need to worry about shadowing by importing at top-level? I assume not if eslint was fine with it but just checking...

Reply:
I checked the node and browser and Meteor shell environments and there weren't any globals or types named `Id` in any of them, so I think we're safe in that regard.

Happy to rename this or restructure this if you have another way to spell this in mind. :)